src/rsa/verification.rs

```diff
@@ -103,10 +121,12 @@ pub fn verify_rsa(params: &RSAParameters,
     try!(bssl::map_result(unsafe {
         GFp_rsa_public_decrypt(decoded.as_mut_ptr(), decoded.len(), n.as_ref(),
                                e.as_ref(), signature.as_ptr(), signature.len(),
-                               params.min_bits, MAX_BITS)
+                               params.min_bits, PUBLIC_MODULUS_MAX_LEN)
     }));
 
-    params.padding_alg.verify(msg, untrusted::Input::from(decoded))
+    untrusted::Input::from(decoded).read_all(
+        error::Unspecified,
+        |m| params.padding_alg.verify(msg, m, n.length_in_bits()))
```

Comment:
Should the closing bracket be on the line below, at the same indentation level as line 127?